src/test/java/org/jenkinsci/plugins/docker/workflow/FromFingerprintStepTest.java

```diff
@@ -54,9 +54,11 @@ public class FromFingerprintStepTest {
         String script = "node {\n" +
             "  sh 'mkdir buildWithFROMArgs'\n" +
             "  writeFile file: 'Dockerfile', text: '" + dockerFile + "'\n" +
+            "  withEnv(['DOCKER_BUILDKIT=0']) {\n" +
```

Comment:
When using BuildKit (which IIUC is currently the default), the built image does not list the FROM image as its parent in `docker inspect` metadata, and the FROM image is not even present in `docker image ls -a`, so the test doesn't work at all. `dockerFingerprintFrom` doesn't run by default as of #180, so I could also see an argument for just deleting the whole test file at this point.